iroha/src/wsv.rs

```diff
@@ -87,6 +90,7 @@ impl WorldStateView {
                 iroha_logger::warn!("Failed to proceed transaction on WSV: {}", e);
             }
             let _ = self.transactions.insert(transaction.hash());
+            task::yield_now().await;
```

Comment:
may be comment out why it should be yielded here? For future reference. Using noop will always be unclear

Reply:
I meant justification why this place is chosen for yielding